<launch>
  <arg name="arena_info_static_topic" value="/arena_info_static" />
  <arg name="arena_info_dynamic_topic" value="/arena_info_dynamic" />

  <!-- <arg name="playground" value = "multi_lane_route_v1.0" /> -->
  <!-- <arg name="playground" value = "tiny_multi_lane_route_v1.0" /> -->
  <!-- <arg name="playground" value = "tiny_multi_lane_route_v4.0" /> -->
  <arg name="playground" value = "tiny_multi_lane_route_v5.0" />

  <node pkg="phy_env_manager" name="phy_env_managing_node" type="phy_env_managing_node" output="screen">
    <param name="vehicle_info_path" type="string" value="$(find playgrounds)/$(arg playground)/vehicle_set.json" />
    <param name="map_path" type="string" value="$(find playgrounds)/$(arg playground)/obstacles_norm.json" />
    <param name="lane_net_path" type="string" value="$(find playgrounds)/$(arg playground)/lane_net_norm.json" />
    <param name="management_rate" type="double" value="500" />
    <remap from="~arena_info_static" to="$(arg arena_info_static_topic)"/>
    <remap from="~arena_info_dynamic" to="$(arg arena_info_dynamic_topic)"/>
  </node>
</launch>
